packages/cardhost/app/controllers/cards.js

```diff
@@ -4,15 +4,20 @@ import { inject as service } from '@ember/service';
 export default class CardsController extends Controller {
   @service cssModeToggle;
   @service cardstackSession;
+  @service router;
 
   get themerClasses() {
-    let editing = this.cssModeToggle.editingCss;
-    if (editing && this.cssModeToggle.isResponsive) {
+    let onThemerRoute = this.router.currentRouteName.includes('themer');
```

Comment:
you could use `this.router.currentRoute.localName === 'themer'`...not sure if that's better though 😄

Reply:
also why not use the getter on line 20, i.e. `this.onThemerRoute`?

Reply:
I think your suggestion is an improvement! I'll fix these two things and merge.